yarn.lock

```diff
@@ -4346,15 +4346,16 @@
     react-is "^16.13.0"
 
 "@react-navigation/core@^6.2.1":
-  version "6.2.1"
-  resolved "https://registry.yarnpkg.com/@react-navigation/core/-/core-6.2.1.tgz#90459f9afd25b71a9471b0706ebea2cdd2534fc4"
-  integrity sha512-3mjS6ujwGnPA/BC11DN9c2c42gFld6B6dQBgDedxP2djceXESpY2kVTTwISDHuqFnF7WjvRjsrDu3cKBX+JosA==
+  version "6.4.0"
```

Comment:
i don't think these changes to the yarn.lock are intended?

Reply:
Something strange is going on with this yarn.lock, if I tried to manually edit the check yarn.lock check is failing.

Reply:
can you just revert your changes for the yarn.lock? you didn't change any dependencies so there's no reason why the yarn.lock file should change i think. maybe something is going on with your local yarn cache?